src/test/java/seedu/address/storage/JsonAdaptedTuteeTest.java

```diff
@@ -0,0 +1,149 @@
+package seedu.address.storage;
```

Comment:
The test cases seem to follow the heuristic: Test invalid inputs individually before combining them. Good Job!